api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirTaskDao.java

```diff
@@ -18,12 +18,14 @@
 import ca.uhn.fhir.rest.param.TokenOrListParam;
 import org.hl7.fhir.r4.model.DomainResource;
 import org.openmrs.module.fhir2.FhirTask;
+import org.openmrs.scheduler.Task;
```

Comment:
This is the wrong Task to import. You should import org.openmrs.module.fhir2.Task.